### DISCUSSION TITLE
New post disappears when date above June

### DISCUSSION THREAD

**Discussion by BethProedrou** [google/docsy#2042] [Q&A]
Hello,
I am having a problem with blog posts. When I create a new post and I set the date for July 2024 or higher the post does not appear in when I run "hugo server". I am new to Hugo so I may be making a mistake somewhere, but I am keeping the posts identical save for their names and dates. I would appreciate any help.

When the date is 2024.06.12 I get the following:
![Screenshot from 2024-06-19 18-54-37](https://github.com/google/docsy/assets/28306623/2317ea59-81f6-480d-9ebe-b0b07ba958ba)

When the date is 2024.07.12 or higher (2024.12.12 in the following screenshot), I get the following:
![Screenshot from 2024-06-19 18-54-15](https://github.com/google/docsy/assets/28306623/90d4a30c-1680-44e2-ad96-20c6114b0a6c)

I am using the latest Hugo, in Linux 22.04.

**Comment by fekete-robert** [comment]
It doesn't appear because you set a publishing date that's in the future.
Either set a date in the past, or run hugo with the "--buildFuture" flag.
